dhcpv4/option_relay_agent_information.go

```diff
@@ -4,6 +4,14 @@ import (
 	"fmt"
 )
 
+type raiValue struct {
+	val []byte
+}
+
+func (rv raiValue) String() string {
+	return fmt.Sprintf("%s (%v)", string([]byte(rv.val)), rv.val)
```

Comment:
It is already a []byte. Does it need the []byte()?

Reply:
right, []byte() is code I forgot to remove, will remove it